cmd/config/internal/commands/run-fns.go

```diff
@@ -4,22 +4,27 @@
 package commands
 
 import (
+	"fmt"
+	"io"
+	"strings"
+
 	"github.com/spf13/cobra"
 	"sigs.k8s.io/kustomize/cmd/config/internal/generateddocs/commands"
+	"sigs.k8s.io/kustomize/kyaml/errors"
 	"sigs.k8s.io/kustomize/kyaml/runfn"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
 // GetCatRunner returns a RunFnRunner.
 func GetRunFnRunner(name string) *RunFnRunner {
 	r := &RunFnRunner{}
 	c := &cobra.Command{
-		Use:     "run DIR",
-		Aliases: []string{"run-fns"},
+		Use:     "run [DIR]",
```

Comment:
A UX question:

Do you see `run --image` command getting more complicated to allow for e.g. mounting volumes? Or at that point, we recommend using `docker run` directly or use a declarative function configuration?

Reply:
I expect it will git a bit more complicate with --network and maybe --volume.  Not much beyond that.  I'd recommend using run for anything that is supported, and using docker to break glass as needed.